docs/integrations/dalex.rst

```diff
@@ -0,0 +1,201 @@
+Dalex Integration
+=================
+
+This integration enables you to log |DALEX| global and local explanations as well as the ``dalex.expainer`` object to Neptune.
+
+.. image:: ../_static/images/others/dalex_neptuneai.png
+   :target: ../_static/images/others/dalex_neptuneai.png
+   :alt: DALEX Neptune integration
+
+Requirements
+------------
+To use Neptune + DALEX integration you need to have installed is |neptune-client| and |neptune-contrib|.
```

Comment:
`you need to have installed is` -> `you need to have installed`

Reply:
done